### PR TITLE
Added lintian overrides for package signature entry

### PR DESCRIPTION
**Pull request by asmorkalov**

### CODE REVIEW

**Comment by asmorkalov** [comment]
@ilya-lavrenov please look at it.

**Comment by ilya-lavrenov** [comment]
i think it's better to:
```
foreach (comp ${ALL_COMPONENTS})
    list(APPEND "${comp_upcase}__LINTIAN_OVERRIDES" "misplaced-extra-member-in-deb")
endforeach()
```

**Comment by asmorkalov** [comment]
Yes, your solution is better. Implemented.

File: cmake/OpenCVPackaging.cmake [hunk 1]
```diff
@@ -289,6 +289,10 @@ else()
     set(TESTS_LINTIAN_OVERRIDES "empty-binary-package") # there is no tests
 endif()
 
+foreach (comp ${ALL_COMPONENTS})
+    list(APPEND "${comp_upcase}__LINTIAN_OVERRIDES" "misplaced-extra-member-in-deb") # for signed packages
```
Comment (ilya-lavrenov) [inline cmake/OpenCVPackaging.cmake]: why do you use `__` instead of `_`?

**Comment by asmorkalov** [comment]
it's copy-paste issue from your code. It's broken too :)

**Comment by ilya-lavrenov** [comment]
> it's copy-paste issue from your code. It's broken too :)

the issue has not been fixed yet. Please, correct the patch.

**Comment by ilya-lavrenov** [comment]
:+1:

**Comment by ilya-lavrenov** [comment]
your patch is still wrong..

**Comment by asmorkalov** [comment]
Sorry for lot of not checked solutions. Finally has done it properly.

**Comment by ilya-lavrenov** [comment]
:+1:

**Pull request merged**